stdlib/public/core/Array.swift

```diff
@@ -1122,15 +1118,16 @@ extension Array: RangeReplaceableCollection {
   internal mutating func _createNewBuffer(
     bufferIsUnique: Bool, minimumCapacity: Int, growForAppend: Bool
   ) {
-    _internalInvariant(!bufferIsUnique || _buffer.isUniquelyReferenced())
-    _buffer = _buffer._consumeAndCreateNew(bufferIsUnique: bufferIsUnique,
-                                           minimumCapacity: minimumCapacity,
-                                           growForAppend: growForAppend)
+    _createNewBuffer(
+      bufferIsUnique: bufferIsUnique,
+      minimumCapacity: minimumCapacity,
+      growForAppend: growForAppend,
+      rangeToNotCopy: 0..<0
+    )
   }
 
   @_alwaysEmitIntoClient
-  @inline(never)
```

Comment:
The original of this was inline(never) for ContiguousArray and unannotated for Array. I've made them match here on the theory that Array is usually the correct one.